shared-data/python/opentrons_shared_data/protocol/models/__init__.py

```diff
@@ -1,8 +1,30 @@
 """Protocol file reading interfaces."""
-from .protocol_schema_v6 import ProtocolSchemaV6
+from .protocol_schema_v6 import (
+    ProtocolSchemaV6,
+    Pipette,
+    Command,
+    Labware,
+    Robot,
+    Metadata,
+    Params,
+    OffsetVector,
+    WellLocation,
+    Module,
+    Location,
+)
 
 
 __all__ = [
     # main interface
     "ProtocolSchemaV6",
+    "Robot",
+    "Labware",
+    "Command",
+    "Pipette",
+    "Metadata",
+    "Params",
+    "OffsetVector",
+    "WellLocation",
+    "Module",
+    "Location",
 ]
```

Comment:
It looks like most of these re-exports aren't being used, and I agree with Max's comment. I know we've got some weirdness still to sort out in `opentrons_shared_data.protocols`, so let's do this for now:

```suggestion
from . import protocol_schema_v6
from .protocol_schema_v6 import ProtocolSchemeV6

__all__ = [
    "ProtocolSchemaV6",
    "protocol_schema_v6",
]
```

Export the main model (`ProtocolSchemaV6`) at the top level, but also re-export the entire `protocol_schema_v6` module for explicitly public access to all the subtypes

Reply:
I can do that but don't we want to change the Robot class name anyway? @mcous it does feel too robust to what's in it